azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -309,7 +322,7 @@ def __ensure_azure_slices_exist(self):
                     for unit_file in files_to_create:
                         self._cleanup_unit_file(unit_file)
 
-            return _AZURE_SLICE if return_azure_slice else "system.slice"
+            logger.info("Agent's CPUQuota: {0}",   self._cgroups_api.get_unit_property(agent_unit_name, "CPUQuotaPerSecUSec"))
```

Comment:
When setting, the property is named CPUQuota (see line 69); when retrieving it is named CPUQuotaPerSecUSec

Reply:
Is this a Cgroups thing?

Reply:
yup; i added a comment in the next iteration of the code; i'll submit the PR later this week